pontoon/base/migrations/0105_unescape_quotes_from_android_strings_dtd_in_translation_memory.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-19 21:19
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import Q, F, Func, Value
+
+
+def unescape_quotes_in_tm_entries_from_android_strings_dtd(apps, schema):
+    """
+    Problem with the synchronization of android_strings.dtd (&quot; and &apos; entities) was fixed in eb5f8d553.
+    However, TM still contains entries with broken entities which break TMX files.
+
+    Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1390111
+    """
+    TranslationMemoryEntry = apps.get_model('base', 'TranslationMemoryEntry')
+
+    def unescape_quotes(fieldname):
+        """Get a SQL function which will unescape quotes in a fieldname."""
+
+        def replace(expr, substring, replacement):
+            """Shortcut for SQL Replace function."""
+            return Func(expr, Value(substring), Value(replacement), function='replace')
+
+        return replace(
+            replace(F(fieldname), '\\u0022', '"'), '\\u0027', "'"
+        )
+
+    TranslationMemoryEntry.objects.filter(
```

Comment:
We should limit the query to `entity__resource__path__contains="mobile/android/base"`, as we did in https://github.com/mozilla/pontoon/commit/4537eac40f36cff404fc36bc9f7a5c4c82b520e8.